packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721BurnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "./AttestationStation.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+/**
+ * @title  Optimist
+ * @dev    Contract for Optimist SBT
+ * @notice The Optimist contract is a SBT representing real humans
+ *         It uses attestations for its base URI and whitelist
+ *         This contract is not yet audited
+ */
+contract Optimist is ERC721BurnableUpgradeable, OwnableUpgradeable, Semver {
+    /**
+     * @notice The attestation station contract where owner makes attestations
+     */
+    AttestationStation public immutable attestationStation;
+
+    /**
+     * @notice The length of the address
+     * used to format token ids into address strings
+     */
+    uint8 private constant _ADDRESS_LENGTH = 20;
```

Comment:
I don't think necessary to make this a constant, can be used inline

Reply:
I like it better as a named constant but don't feel strongly